tests/storage_test.go

```diff
@@ -0,0 +1,260 @@
+package inttest
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/asecurityteam/asset-inventory-api/pkg/domain"
+	"github.com/asecurityteam/asset-inventory-api/pkg/storage"
+	"github.com/asecurityteam/logevent"
+	"github.com/asecurityteam/runhttp"
+	"github.com/asecurityteam/settings"
+	packr "github.com/gobuffalo/packr/v2"
+	"github.com/stretchr/testify/assert"
+)
+
+var db *sql.DB
+var dbStorage *storage.DB
+var ctx context.Context
+
+func TestMain(m *testing.M) {
+	// make sure the DB is cleaned of all rows
+	ctx = context.Background()
+	ctx = logevent.NewContext(ctx, stdoutLogger())
+	source, err := settings.NewEnvSource(os.Environ())
+	if err != nil {
+		panic(err.Error())
+	}
+
+	postgresConfigComponent := &storage.PostgresConfigComponent{}
+	dbStorage = new(storage.DB)
+	if err = settings.NewComponent(ctx, source, postgresConfigComponent, dbStorage); err != nil {
+		panic(err.Error())
+	}
+
+	db, err = connectToDB()
+	if err != nil {
+		panic(err.Error())
+	}
+
+	// if err = dropTables(db); err != nil {
```

Comment:
will these if blocks be used eventually?

Reply:
I've deleted that comment.